[JENKINS-30613] Code clean-up

pjanouse opened the pull request:
Please review, thx.

pjanouse committed:
[JENKINS-30613] Code clean-up

KostyaSha commented on test/src/test/java/hudson/cli/DeleteJobCommandTest.java:
```diff
@@ -33,7 +33,9 @@
 import java.io.IOException;
 
-import static hudson.cli.CLICommandInvoker.Matcher.*;
+import static hudson.cli.CLICommandInvoker.Matcher.failedWith;
```
What does this fixes?

jtnord replied:
static import something.* is discouraged except for org.hamcrest and org.junit?

KostyaSha commented on core/src/main/resources/hudson/cli/Messages.properties:
```diff
@@ -81,6 +81,5 @@ Cannot build {0} because its configuration has not been saved.
 
-DeleteNodeCommand.ShortDescription=\
- Deletes node(s)
+DeleteNodeCommand.ShortDescription=Deletes node(s)
```
Could you describe what is the difference? Because previous line also has new lined message.

oleg-nenashev commented:
This PR does not provide a significant added value, but it's harmless.
LGTM

oleg-nenashev commented:
Noting is not required IMHO

KostyaSha commented:
I see no answer for difference for properties and there is no good description in issue or in commit message.

pjanouse commented:
Hello,

static import with wildcard isn't discouraged, but from my PoV it isn't the best approach. I've been avoiding it if meaningful. It was introduced by me accidentally with JENKINS-28041. I've realized that during my work on JENKINS-30521, where was correctly mentioned that this correction isn't related to JENKINS-30521, so I've created another JIRA for this clean-up.

Anyway, KostyaSha noted why I've put a new line to the Messages.properties item. I don't have any argument to support my original impl. so I fixed that. I've created the same in impl. for JENKINS-28041, so I've corrected that as well. That's it.

KostyaSha commented:
Thanks, because it seems that you followed existed code when added new, but now decided to revert. And it looks weird because other file entries with new lines. That's why interested about goal of changes.
